.travis.yml

```diff
@@ -24,6 +27,8 @@ env:
     - TESTENV=doctesting
     - TESTENV=py27-cxfreeze
     - TESTENV=coveralls
+  allow_failures:
+    - TESTENV=py35
```

Comment:
One of the problem, is AFAIK, travis 3.5 is still on early alpha that do not have the Change in the AST.